interface/modules/custom_modules/oe-module-weno/templates/synch.php

```diff
@@ -141,7 +141,7 @@ function downloadWenoLogCsvAndZip()
         RecursiveIteratorIterator::LEAVES_ONLY
     );
 
-    foreach ($files as $file) {
+    foreach ($files as $name => $file) {
```

Comment:
$name not used, so revert this change. Guessing this is why rector is failing test.